dev/archery/archery/integration/datagen.py

```diff
@@ -1547,6 +1567,12 @@ def _temp_path():
         .skip_category('Go')
         .skip_category('Rust'),
 
+        # TODO: Dictionary unsigned indices
```

Comment:
Why is it marked TODO? Do you intend to add specific per-language comments instead?